email-log.php

```diff
@@ -76,26 +128,41 @@ function __construct() {
 		load_plugin_textdomain( 'email-log', false, $this->translations );
 
 		// Register hooks
+
 		add_action( 'admin_menu', array( $this, 'register_settings_page' ) );
 
 		// Register Filter
+
```

Comment:
We may have to remove the extra lines that got added. Apart from this everything else looks perfect.